index.html

```diff
@@ -72,20 +72,22 @@ <h1 id="big-timer">
         <div class="center">
           <h3 class="head-line">The Projects</h3>
           <div class="text-container center">
-            Our partners have offered up a set of challenges for you to solve. Below you'll see some of the things you might be buildling during Green Hack (of course, you're welcome to go rogue if you have a great idea with impact).
+            Our partners have offered up a set of challenges for you to solve. Below you'll see some of the things you might be building during Green Hack (of course, you're welcome to go rogue if you have a great idea with impact).
           </div>
         </div>
 
-        <div class="charities">
-          <section class="text-container charity">
+        <div class="projects">
+          <section class="text-container proj">
```

Comment:
I reckon we could probably name this class something more descriptive than `proj`. Not entirely sure what it could become at a glance though...